content/md/en/docs/reference/command-line-tools/srtool.md

```diff
@@ -1,9 +1,409 @@
 ---
 title: srtool
-description: Command-line reference information for the srtool program.
+description: Describes the Docker container and command-line interface for building the Substrate WebAssembly runtime.
 keywords:
 ---
 
-The `srtool` allows building WASM runtimes in a deterministic way, allowing CIs and users to produce a strictly identical WASM runtime.
+The Substrate runtime tool (`srtool`) is a Docker container that enables you to build the Substrate WebAssembly runtime in a deterministic way.
+By using this tool, you can ensure that the same source code consistently reproduces an identical WebAssembly blob. 
+You can also use the tool to inspect and audit the runtime for any Substrate-based chain and to integrate building the WebAssembly runtime into your CI/CD pipeline.
 
-- [Go to documentation](https://github.com/paritytech/srtool).
+## Get the Docker image
+
+You must have a Docker account and Docker installed in your build environment to access the Substrate runtime tool container.
+If you sign in to the Docker Hub, you can search for the `paritytech/srtool` container and corresponding tagged images that are available there.
+
+### Image naming convention
+
+The naming convention for `paritytech/srtool` Docker images specifies the version of the Rust compiler used to compile the code included in the image.
+For example, the image `paritytech/srtool:1.62.0` indicates that the tools included in the image were compiled with version `1.62.0` of the `rustc` compiler.
+In addition to the compiler version, some images specify the version of the build script used.
+For example, an image named `paritytech/srtool:1.62.0-0.9.19` was compiled with version `1.62.0` of the `rustc` compiler but using the version `0.9.19` of the build script.
+
+The image that only specifies the compiler version always contains the latest version of the software.
+
+### Download an image from the Docker Hub
+
+You can pull a  `paritytech/srtool` container image directly from Docker Hub or install and use one of the [helper programs](#get-helper-programs) for working with the  `paritytech/srtool` container.
+
+To pull the image from the Docker Hub:
+
+1. Sign in to Docker Hub.
+
+2. Type `paritytech/srtool` in the Search field and press Enter.
+
+3. Click **paritytech/srtool**, then click **Tags**.
+
+4. Copy the command for the image you want to pull.
+   
+5. Open a terminal shell on your local computer.
+   
+6. Paste the command you copied from the Docker Hub.
+   
+   For example, you should run a command similar to the following:
+
+   ```bash
+   docker pull paritytech/srtool:1.62.0
+   ```
+
+   The command downloads and unpacks the image.
```

Comment:
This is precisely the reason why I recommend using `srtool-cli`, the whole thing you describe can then be done as:
```
srtool pull
```